src/cbmc/bmc_cover.cpp

```diff
@@ -309,6 +311,42 @@ bool bmc_covert::operator()()
   for(const auto & it : goal_map)
     if(it.second.satisfied) goals_covered++;
   
+  if (bmc.options.get_bool_option("gen-c-test-case"))
+  {
+    size_t test_case_no=0;
+    for(auto& test : tests)
+    {
+      c_test_case_generatort gen(get_message_handler());
+      std::vector<std::string> goal_names;
+      for(const auto& goalid : test.covered_goals)
+        goal_names.push_back(
+                             as_string(goal_map.at(goalid).description)
```

Comment:
As in this line should read like:

    goal_names.push_back(as_string(goal_map.at(goalid).description)+"\n *  "
      +id2string(goal_map.at(goalid).source_location.get_file())
      +":"
      +id2string(goal_map.at(goalid).source_location.get_line()));